Build for Ubuntu bionic

When building for bionic there are some compiler errors:

```
backend/go.c:82:5: error: narrowing conversion of '-1' from 'int' to 'mftype {aka unsigned int}' inside { } [-Wnarrowing]
     };
     ^
```

When adding `CXXFLAGS=-Wno-narrowing`, then we have:

```
backend/code.h:536:17: error: expected initializer before ‘cat’
 code * __pascal cat (code *c1 , code *c2 );
                 ^~~
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -18,7 +18,9 @@ env:
     global:
         # Make sure beaver is in the PATH
         - PATH="$(git config -f .gitmodules submodule.beaver.path)/bin:$PATH"
+    matrix:
         - DIST=xenial
+        - DIST=bionic
 
 install: beaver install
 

--- a/beaver.Dockerfile
+++ b/beaver.Dockerfile
@@ -1,1 +1,1 @@
-FROM sociomantictsunami/base:v5
+FROM sociomantictsunami/base:v6-test

